src/Apps/Order/OrderApp.tsx

```diff
@@ -22,7 +29,12 @@ export interface OrderAppProps {
   router: Router
 }
 
-export class OrderApp extends React.Component<OrderAppProps> {
+interface OrderAppState {
+  stripe: stripe.Stripe
```

Comment:
Not a problem with this PR: kind of a bummer that the stripe typings are exposed through a global namespace rather than through the module system 😕

Reply:
I guess it's meant to reflect the way it's imported

Reply:
Ah, true, it's defined on `window`. Still, a global namespace assumes that all typescript files in one's project will be running in a browser env that has stripe available. Unlikely to cause problems though.

Reply:
yeah, jest's types have the same issues, I think you can start writing `expect` anywhere:

<img width="945" alt="screen shot 2018-08-28 at 8 04 33 am" src="https://user-images.githubusercontent.com/49038/44721785-05deb600-aa99-11e8-9993-141e1b4a6d96.png">